src/redux/books/books.js

```diff
@@ -1,48 +1,60 @@
-import { v4 as uuidv4 } from 'uuid';
+import { createAsyncThunk } from '@reduxjs/toolkit';
 
 // Actions
+const GET_BOOKS = 'bookstore/books/GET_BOOKS';
 const ADD_BOOK = 'bookstore/books/ADD_BOOK';
 const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
 
-const defaultState = [
-  {
-    id: uuidv4(),
-    title: 'Book title',
-    author: 'Author',
-  },
-  {
-    id: uuidv4(),
-    title: 'Book title',
-    author: 'Author',
-  },
-  {
-    id: uuidv4(),
-    title: 'Book title',
-    author: 'Author',
-  },
-];
+const defaultState = [];
 
 // Reducer
 const booksReducer = (state = defaultState, action) => {
   switch (action.type) {
+    case GET_BOOKS:
+      return action.payload;
     case ADD_BOOK:
       return [...state, action.payload];
     case REMOVE_BOOK:
-      return state.filter((book) => book.id !== action.payload.id);
+      return [...state.filter((book) => book.id !== action.payload.id)];
```

Comment:
- [ ] When i try to remove your book, your app successfully removes the book from the API 😊 But the problem is after clicking on the remove button i always need to refresh the page to remove that book from the DOM. Kindly fix this issue 👍 to make your application respond like a single page application 🤔

Reply:
@HAMMAS-SALEEM please can you hint how to remove it from the DOM too? Thank you